docs/howtos/fhir-conversions.rst

```diff
@@ -0,0 +1,64 @@
+FHIR Conversions
```

Comment:
**Docs**
@cmungall I wanted some docs that were easily accessible / in the main docs rather than the code, but I couldn't find a good spot, so I chose here. The main thing I wanted to document was the config file. I was originally going to do `docs/converters/obo-graph-to-fhir.rst`, but the config file isn't actually used by that converter; it's used by `dump()` in CLI.py.